include/cuspreadinterp.h

```diff
@@ -1,8 +1,7 @@
+#ifndef __CUSPREADINTERP_H__
```

Comment:
Again, this could probably stay in `src/`.

Reply:
See 5ab18e4.